web/packages/teleport/src/Discover/Database/common.tsx

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Box, Label as Pill, Text } from 'design';
+import * as Icons from 'design/Icon';
+
+import { AgentLabel } from 'teleport/services/agents';
+import { LabelsCreater, TextIcon } from 'teleport/Discover/Shared';
+
+export const Labels = ({
+  labels,
+  setLabels,
+  disableBtns = false,
+  dbLabels,
+  showLabelMatchErr = false,
+  autoFocus = true,
+}: {
+  labels: AgentLabel[];
+  setLabels(l: AgentLabel[]): void;
+  disableBtns?: boolean;
+  dbLabels: AgentLabel[];
+  showLabelMatchErr?: boolean;
+  autoFocus?: boolean;
+}) => {
+  const hasDbLabels = dbLabels.length > 0;
+  return (
+    <Box mb={2}>
+      <Text bold>Optionally Define Matcher Labels</Text>
+      <Text typography="subtitle1" mb={2}>
+        {!hasDbLabels && (
+          <>
+            Since no labels were defined for the registered database from the
+            previous step, the matcher labels are defaulted to wildcards which
```

Comment:
i didn't think this was necessary since we show the user (it'll just be readonly so they can't change it for this case):

<img width="613" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/e014a66d-cf3d-4aa9-92d4-7e370eac0a57">